front_matter/mod.ts

```diff
@@ -115,9 +114,11 @@
  *
  * @module
  */
-import type { extract as extractJson } from "./json.ts";
-import type { extract as extractToml } from "./toml.ts";
-import type { extract as extractYaml } from "./yaml.ts";
+import { extract as extractJson } from "./json.ts";
+import { extract as extractToml } from "./toml.ts";
+import { extract as extractYaml } from "./yaml.ts";
 
 export * from "./create_extractor.ts";
 export * from "./test.ts";
+
+export { extractJson, extractToml, extractYaml };
```

Comment:
For reviewers: it looks like exporting these is needed to get the links like `{@linkcode extractJson | extract}` used in the module level doc to work correctly.